src/reference/90-Developers-Guide/02-Coding-Guideline/00.md

```diff
@@ -175,13 +179,13 @@ object FooID {
 }
 ```
 
-#### Implicit converter for syntax (enrich-my-library) should be imported
+#### Implicit conversions for syntax (enrich-my-library pattern) should be imported
```

Comment:
I think your distinction makes semantic sense, I just find the phrase "implicit conversion" used interchangeably in this manner across pretty much all Scala documentation I've read, and I've never seen "implicit converter" used… https://www.google.co.th/search?q=scala+implicit+converter